tests/unit/mock-data.json

```diff
@@ -35,6 +35,96 @@
       "name": "Hedwig",
       "owner": "Harry Potter",
       "species": "Owl"
+    },
+    {
+      "id": "8",
+      "name": "pip (deceased)",
+      "owner": "Ian Wilson",
+      "species": "Pea Puffer"
+    },
+    {
+      "id": "10",
+      "name": "Benny (deceased)",
+      "owner": "Ian Wilson",
+      "species": "Pea Puffer"
+    },
+    {
+      "id": "9",
+      "name": "pip3 (planned)",
+      "owner": "Ian Wilson",
+      "species": "Pea Puffer"
+    },
+    {
+      "id": "7",
+      "name": "Qwerty",
+      "owner": "Elliot Anderson",
+      "species": "Beta fish"
+    },
+    {
+      "id": "11",
+      "name": "Qwerty",
+      "owner": "Mr. Robot",
+      "species": "Beta fish"
+    },
+    {
+      "id": "12",
+      "name": "Ollie",
+      "owner": "David's Roommate",
+      "species": "Cat"
+    },
+    {
+      "id": "13",
+      "name": "Sage",
+      "owner": "Ian Wilson",
+      "species": "Dog"
+    },
+    {
+      "id": "14",
+      "name": "Goose",
+      "owner": "Ian Wilson",
+      "species": "Dog"
+    },
+    {
+      "id": "15",
+      "name": "Wilson",
+      "owner": "Tom Hanks",
+      "species": "Volleyball"
+    },
+    {
+      "id": "16",
+      "name": "Clifford",
+      "owner": "Emily Elizabeth",
+      "species": "Big Red Dog™"
+    },
+    {
+      "id": "17",
+      "name": "Toothless",
+      "owner": "Ian's old roommate",
+      "species": "axolotl"
```

Comment:
axolotl -> Aaxolotl?

Reply:
It is [axolotl](https://en.wikipedia.org/wiki/Axolotl), believe it or not

Reply:
Oh, actually I mean Capitalized hahaha.